compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/DisposableSaveableStateRegistry.skiko.kt

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
This looks very similar to the Android version. Do you plan to commonize it? If not, what are the differences?

Reply:
Not yet. It relies on platform specific representation that we don't have